openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeRulesDefinition.java

```diff
@@ -50,45 +50,49 @@ public void define(Context context) {
     // Manually created rules for compiler warnings
     NewRule warning = repository.createRule(COMPILER_WARNING_RULEKEY).setName("Compiler warnings").setSeverity(
         Priority.CRITICAL.name());
+    warning.setTags("compiler.warnings");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "compiler.warnings" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1192)